scripts/prepare.sh

```diff
@@ -102,7 +102,19 @@ function update_podspec() {
 
 update_cartage_binary_project_spec $cartage_spec_OpenTelemetryApi $version
 
-sha1=$(shasum -a 1 artifacts/OpenTelemetryApi.xcframework.zip | awk '{print $1}')
+sha1=$(shasum -a 1 artifacts/OpenTelemetryApi.zip | awk '{print $1}')
 update_podspec $podspec_OpenTelemetryApi $version $sha1
 
 commit_push
+
+echo "Checking if the github release $version exists"
```

Comment:
had to move back to prepare else lint would fail. For now it is okay but would like not to publish version in prepare.